src/ol_infrastructure/applications/airbyte/airbyte_server_policy.hcl

```diff
@@ -0,0 +1,35 @@
+path "secret-operations/global/odl_wildcard_cert" {
+  capabilities = ["read"]
+}
+
+path "secret-data/data-ci/airbyte/*" {
+  capabilities = ["read"]
+}
+
+path "secret-data/data-qa/airbyte/*" {
+  capabilities = ["read"]
+}
+
+path "secret-data/data-production/airbyte" {
```

Comment:
We have separate Vault clusters for each environment level, so there's no need to include the env-specific paths in the policy definition. We should aim to keep these as flat as possible.

Reply:
Maybe even mount a backend specific to Airbyte so that we can scope things easily (e.g. `secret-airbyte`)

Reply:
The first thing is addressed, the second comment not yet.